### DISCUSSION TITLE
유저 피드백 들어왔습니다. 의견 달아주세요~

### DISCUSSION THREAD

**Discussion by hoetaek** [teacher-coder/picture-muvie#96] [Ideas]
1. 가사 찾기 할 때 '동요'는 검색이 안되는 경우가 있는 듯 합니다. 혹시 검색하면 목록 중에서 찾을 수 있는 기능 같은게 있으면 좋을 것 같습니다. (ex 모두 다 꽃이야 검색하면 다른 노래가사가 나옵니다.)

2. 가사 압축하기 기능이 완전 좋은데,, 반대로 '가사 분할하기' 기능도 있으면 좋겠습니다. (넌 할 수 있어라고 말해주세요를 하면 12명 짜리가 만들어집니다...)

3. 완성되는 HWP 파일에서 가사 크기를 좀 손쉽게 편집하는 방법이 있으면 좋겠습니다. 글씨 크기를 저는 좀 크게 하는 걸 좋아하는 편인데 생각보다 가사 글씨가 작게 나오는 것 같습니다..

**Comment by hoetaek** [comment]
1번에 대해서 나도 동요 검색 안되더라..ㅋㅋㅋ 모두 다 꽃이야 동요라고 검색하니까 오히려 에러 발생해서 이슈 달아둠 #94

**Reply by hoetaek** [reply]
이상한 게 네이버에 직접 검색했을 때는 바로 동요 가사가 떠ㅋㅋㅋ

**Reply by Ridealist** [reply]
동요 검색은 동요 가사 제공해주는 사이트를 찾아봐야 될 것 같아. 멜론 지니 뮤직 이런 곳에서 동요가사가 많이 없어서 그런듯?

**Reply by Ridealist** [reply]
무료 api라 뭔가 네이버 검색 기능 전체를 제공해주지는 않는 느낌이야. 맞어 검색 결과가 좀 빈약하더라고.

**Reply by hoetaek** [reply]
어떻게 보완할 수 있을까ㅠㅠ

**Reply by Ridealist** [reply]
일단 지금 로그를 보는 중. 검색이 아예 안되는건지 아니면 다른 걸 스크랩 해오는 건지 보려고.

**Comment by hoetaek** [comment]
2번에 대해서도 내가 작성한 이슈 #88 을 일단 적용하는 걸로?

**Reply by Ridealist** [reply]
오키 이건 바로 적용 가능하니까 지금 해결해볼게 내가~

**Reply by Ridealist** [reply]
응? 회택이 너가 self-assignee 달아놓은게 너가 작업하고 있어서 그런건가? @hoetaek

**Reply by hoetaek** [reply]
내가 오늘 할게!ㅋㅋㅋㅋ

**Reply by hoetaek** [reply]
가사 분할하기는 또 내가 카드 뷰 적용하면 괜찮을 거 같아
#38

**Reply by Ridealist** [reply]
오키도키~ 난 1번 부터 해결하고 있을게!

**Comment by hoetaek** [comment]
3번은 어떻게 할지 일단 고민해보자! 폰트 설정까지 가능하게 할 것인가...
너무 세부적인 조정은 좀 그렇고 "크게 - 보통 - 작게" 이렇게 세개 중에 고르면 어떨까 싶어

**Reply by Ridealist** [reply]
응 3개 중에 선택하는 거 좋다.
그대신 "크게" 선택하면은 가사 길이가 좀만 길어져도 2줄로 나눠야 되서
한 줄로 맞추려면 최대 가사 길이랑 연동해서 기능을 제공해야 될것 같아
